packages/nimble-components/src/table/index.ts

```diff
@@ -63,6 +68,7 @@ export class Table<
     private readonly table: TanStackTable<TData>;
     private options: TanStackTableOptionsResolved<TData>;
     private readonly tableValidator = new TableValidator();
+    private notifiers: Notifier[] = [];
```

Comment:
not: think I'd suggest naming this columnNotifiers just to emphasize this is intended just for TableColumns